builders/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,328 @@
+
+### Define Custom Theme Colors
+
+RainbowKit offers three built-in theme functions: `lightTheme`, `darkTheme`, and `midnightTheme`. These theme functions return a theme object, which you can pass into the `RainbowKitProvider` prop `theme` to set custom colors, border radius, font stack, and overlay blur. Update `providers.tsx` with the following code. Be sure to add `darkTheme` to the `@rainbow-me/rainbowkit`import statement to allow your changes to render correctly.
+
+```js title="providers.tsx"
+--8<-- 'code/builders/integrations/wallets/rainbowkit/providers.tsx'
+```
+
+## Handle Disconnections
+
+You can now disconnect MetaMask from your dApp and then reconnect to test your customizations. There are two options for completing this step.  
+
+### Disconnect from DApp {: #disconnect-from-dapp }
+
+RainbowKit includes a **Disconnect** button out of the box. To open the modal, select the arrow next to your account number. Click the **Disconnect** button. You should now see **Connect Wallet**; your account information should no longer be visible. 
+
+![Built in Disconnect button](/images/builders/integrations/wallets/rainbowkit/rainbowkit-2.webp)
+
+### Disconnect from MetaMask Mobile {: #disconnect-from-metamask-mobile }
+
+Some users prefer to disconnect from their mobile wallet rather than use a button within a dApp. To use this method: 
+
+1. Select the MetaMask extension in your browser to open the modal
+2. Select the three dots in the upper right corner of the MetaMask modal
+3. Select **Connected sites**
+4. Review the list of sites connected to your wallet
+5. Select **Disconnect** for each site you want to disconnect
+
+## Final Result {: #final-result }
+
+The **Connect Wallet** button on your home page should now render in the color you entered for `accentColor` when customizing the theme. When you click **Connect Wallet**, you will see the same accent color in use. Select MetaMask and sign the transaction to authorize the connection. You should now see **Moonbase Alpha** as the connected network and your DEV token balance for the account balance without manually switching networks. 
+
+![Theme customization on the user modal](/images/builders/integrations/wallets/rainbowkit/rainbowkit-3.webp)
+
+This guide includes only a few of the customization options available through RainbowKit. You can learn more about the capabilities and options of this library by visiting [RainbowKit Docs](https://www.rainbowkit.com/docs/introduction){target=\_blank}.
+
+You can view the complete example code in the [rainbow-manual-build-demo repository](https://github.com/papermoonio/rainbowkit-manual-build-demo){target=\_blank}
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
```

Comment:
```suggestion
--8<-- 'text/_disclaimers/third-party-content.md'
```

The disclaimer you had here is used for the introduction. You can use this one instead for the end of the page